app/lib/service/entrypoint/search.dart

```diff
@@ -72,3 +73,11 @@ Future _main(FrontendEntryMessage message) async {
 
   await runHandler(_logger, searchServiceHandler);
 }
+
+Future _worker(WorkerEntryMessage message) async {
+  message.protocolSendPort.send(WorkerProtocolMessage());
+  await popularityStorage.start();
+  setupSearchPeriodicTasks();
+  unawaited(searchBackend.updateSnapshotInForeverLoop());
+  await Completer().future; // never completes
```

Comment:
Just curious, why not do?
```suggestion
  await searchBackend.updateSnapshotInForeverLoop();
```